math/Magic Square

```diff
@@ -0,0 +1,54 @@
+#include <bits/stdc++.h>
```

Comment:
Please remove this header because it is quite linux-specific and increases the compile time.